File: Python/average.py
# def average(num_list):
#   mean = sum(num_list) / len(num_list)
#   return mean

# print(average([1, 2, 5, 75, 12, 43]))


from functools import reduce

def average(num_list):
  total = reduce(lambda a,b : a+b,num_list) / len(num_list)
  return total
  
print(average([1, 2, 5, 75, 12, 43]))

File: Javascript/average.js
// const arrAvg = arr => arr.reduce( ( a, b ) => a + b, 0 ) / arr.length;
    
// console.log(arrAvg([6, 8, 4, 32, 14]))

function arrAvg(num) {
  let sum = 0;
  if(!num.length){
    return sum;  
  }
  for (let i = 0; i < num.length; i++){
      sum += num[i];
  }
  return sum / num.length;
}
console.log(arrAvg([6, 8, 4, 32, 14]))